test/ptp/ptp.go

```diff
@@ -156,7 +156,11 @@ var _ = Describe("[ptp]", func() {
 						ptpRunningPods = append(ptpRunningPods, pod)
 					}
 				}
-				Expect(len(ptpRunningPods)).To(BeNumerically(">=", 2), fmt.Sprint("Fail to detect PTP slave/master pods on Cluster"))
+				expectedPtpRunningPods := 2
+				if !discovery.Enabled() {
+					expectedPtpRunningPods = 1
```

Comment:
Shouldn't be the other way around? 1 pod in discovery mode (only slave).
Also, I think it would make sense to check we found a slave splitting the or above.

Reply:
Oh yes you are right. Changed. Please review it once again